### PR TITLE
Setup ddns when bridge is none

### PR DESCRIPTION
**Pull request by ntkathole**

### CODE REVIEW

**Comment by lpramuk** [comment]
Any reasoning behind the change?

**Comment by ntkathole** [comment]
> Any reasoning behind the change?
> 
> What are those machines having no bridge?

@lpramuk This change is required for some jobs where we install satellite and also create vm which should use have resolvable hostname. When we use product_install  task with create_vm=true, internally it calls vm_create task and bridge is not passed so by default it will set value `bridge=br0`. In such case we want ddns task to run.

File: automation_tools/__init__.py [hunk 1]
```diff
@@ -1321,7 +1321,7 @@ def vm_create():
     if (
         options['bridge'] in ['br0', 'foreman'] and
         'DDNS_HASH' in os.environ and 'DDNS_PACKAGE_URL' in os.environ
-    ):
+    ) or os.environ.get('BRIDGE') is None:
```
[the remark this answers is not in the document]
Comment (ntkathole) [reply automation_tools/__init__.py]: @lpramuk makes sense. Updated.

**Comment by ntkathole** [comment]
@lpramuk please revisit!

**Comment by ntkathole** [comment]
Bump!

**Comment by ntkathole** [comment]
@lpramuk does it look good to you?

**Comment by ntkathole** [comment]
Client automation and custom_cert automation job is failing due to this.

**Pull request merged**